tests/snuba/search/test_backend.py

```diff
@@ -1159,12 +1175,9 @@ def __eq__(self, other):
             sort_by='new',
         )
         assert query_mock.call_args == mock.call(
-            orderby=['-first_seen', 'issue'],
-            aggregations=[
-                ['toUInt64(min(timestamp)) * 1000', '', 'first_seen'],
-                ['uniq', 'issue', 'total'],
-            ],
-            having=[['first_seen', '>=', Any(int)]],
+            aggregations=[['count()', '', 'times_seen'], ['uniq', 'issue', 'total']],
```

Comment:
@wedamija do these actually mean the same thing? it's looking fairly suspect to me. do you have visibility to what I'm seeing here?

Reply:
I think one is count of events, and one is count of issues?